4.5.2 release (#149)
blackbaud/skyux-popovers: CHANGELOG.md

# 4.5.2 (2021-05-10)

> - Fixed the dropdown component to focus on the first menu item when clicking the dropdown button. [#148](https://github.com/blackbaud/skyux-popovers/pull/148)

Reviewer: ```suggestion
- Fixed the dropdown component to focus on the first menu item after users select the dropdown button. [#148](https://github.com/blackbaud/skyux-popovers/pull/148)
```
